src/ext/h1_reason_phrase.rs

```diff
@@ -147,7 +147,7 @@ const fn is_valid_byte(b: u8) -> bool {
     //
     // The 0xFF comparison is technically redundant, but it matches the text of the spec more
     // clearly and will be optimized away.
-    #[allow(unused_comparisons)]
+    #[allow(unused_comparisons, clippy::absurd_extreme_comparisons)]
```

Comment:
Why does clippy think this is absurd? They look like valid comparisons to me...

Reply:
clippy seems to treat comparisons with values beyond what the type can represent as absurd.

https://rust-lang.github.io/rust-clippy/rust-1.78.0/index.html#/absurd_extreme_comparisons